### ISSUE TITLE
MAPPINGS CALIBRATION: the mapping GIF sheets in SemioBrain Database need to be finalised (final checks to be done after #82 or #135)

### ISSUE THREAD

**Issue by thenineteen** [thenineteen/Semiology-Visualisation-Tool#75]
Due to the "postcode" method of data collection (!) where each brain subregion (london) data gets carried forward to all the hierarchical levels (Greater London, England, UK, Europe), this can create duplicate data when being mapped on the excel sheets (named "GIF TL").

e.g. I've discovered that two subregions of

```
STG (includes Transverse Temporal Gyrus, Both Planum) 

```

namely:

```
Transverse Temporal Gyrus (Heschl's, BA 41,  42, ?opercula) 
Planum Temporale
```

had double map on the STG. 

So if a patient had planum polare localisation, they would get a 1 here and 1 at STG. when mapped, they would get mapped to planum polare gifs 185 and 186, but also 201, 202 (STG) and 185 and 186 again - effectively getting doubling their localisation scores for planum polare.

The problem with the post code system, (which I've always maintained complicates analysis), is:

1. duplication of data as above - see below to try to mitigate this risk
2. blurring and adding data (e.g. STG regions not part of planum polare)

point 1 is the most pressing issue. 

There are two things we need to do :

1. can I request @fepegar to create a "gifs" tab where we simply click on a check box of a gif number and name (e.g. " Right STG superior temporal gyrus - 201") and we see the gif on the brain in 3D? This will help @neurokleos  and I to calibrate our mappings more correctly.

second one in another issue : #76

**Comment by fepegar** [comment]
This should be doable. You'll need to define exactly what you mean by "see the gif on the brain in 3D".

**Comment by fepegar** [comment]
I'm going to move the Slicer request to a different issue, I hope that's ok.

**Comment by fepegar** [comment]
Shall we close this issue and keep discussing on #76 and #82?

**Comment by thenineteen** [comment]
cool

when we can visualise gifs (#82) then we can properly calibrate the mapping tabs in the excel file. so this will still be needed

**Comment by thenineteen** [comment]
data as is

tonic left:
![tonic left](https://user-images.githubusercontent.com/10786099/86423160-fce34f00-bcd6-11ea-99eb-728c18441e5a.png)





asymmetric tonic left:
![asymmetric tonic left](https://user-images.githubusercontent.com/10786099/86423163-02d93000-bcd7-11ea-9db6-e84e9f72397f.png)


other complex motor :
![other complex motor](https://user-images.githubusercontent.com/10786099/86423172-0b316b00-bcd7-11ea-8f3b-33994089ae7f.png)

**Comment by thenineteen** [comment]
#76 was solved and closed (using a Hierarchy Reversal Class and Dictionary)
#82 is open, however, if #135 is solved, then #82 is not relevant any longer and this Mapping Calibration can undergo final checks

**Comment by thenineteen** [comment]
#137 now needs a fix to continue mappings calibrations

**Comment by thenineteen** [comment]
- [ ] calibrated TPO J in 5677bd020d4d6772cbb762f68af5ae33b5c49999

**Comment by thenineteen** [comment]
- [ ] GIF TL Sheet calibration complete

**Comment by thenineteen** [comment]
for the mapping of the rolandic operculum (low BA4) we had the precentral gyrus and frontal operculum.

look at articles, it is best to also include the GIF parecellations named central operculum (GIF numbers 113 and 114)

e.g. see non scientific article image from the BBC in 2002:
![image](https://user-images.githubusercontent.com/10786099/91106135-d8b64580-e669-11ea-91ba-0b4a17129e36.png)


cf

![image](https://user-images.githubusercontent.com/10786099/91106119-d05e0a80-e669-11ea-85a3-47d66b7ef37e.png)

**Comment by thenineteen** [comment]
opercular part of IFG: we also used this as a label for the opercula, so will leave it as including IFG par opercularis, frontal operculum and central operculum (technically at least shouldn't include the latter but due to the data collection labelling, the majoriy of opercular labels get lumped into this category, it should include all the above 3 mappings)
